packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -379,175 +396,172 @@ const FinancialAssessment = observer(() => {
-                    handleSubmit,
-                    isSubmitting,
-                    setFieldTouched,
-                    dirty,
-                    setFieldValue,
-                }) => (
-                    <React.Fragment>
-                        {!is_appstore && isMobile() && is_confirmation_visible && (
-                            <ConfirmationPage toggleModal={toggleConfirmationModal} onSubmit={handleSubmit} />
-                        )}
-                        {(isDesktop() || is_appstore) && (
-                            <ConfirmationModal
-                                is_visible={is_confirmation_visible}
-                                toggleModal={toggleConfirmationModal}
-                                onSubmit={handleSubmit}
-                            />
-                        )}
-                        <LeaveConfirm onDirty={isMobile() ? showForm : () => undefined} />
-                        {is_form_visible && (
-                            <form className='account-form account-form__financial-assessment' onSubmit={handleSubmit}>
-                                {is_mf && is_financial_information_incomplete && !is_submit_success && (
-                                    <div className='financial-banner'>
-                                        <div className='financial-banner__frame'>
-                                            <div className='financial-banner__container'>
-                                                <Icon icon='IcAlertWarning' />
-                                                {isMobile() ? (
-                                                    <Text size='xxxs' line_height='s'>
-                                                        <Localize i18n_default_text='To enable withdrawals, please complete your financial assessment.' />
-                                                    </Text>
-                                                ) : (
-                                                    <Text size='xxs' line_height='l'>
-                                                        <Localize i18n_default_text='You can only make deposits at the moment. To enable withdrawals, please complete your financial assessment.' />
-                                                    </Text>
-                                                )}
-                                            </div>
+        <Formik initialValues={setInitialFormData()} enableReinitialize validate={validateFields} onSubmit={onSubmit}>
+            {({
+                values,
+                errors,
+                status,
+                touched,
+                handleChange,
+                handleBlur,
+                handleSubmit,
+                isSubmitting,
+                setFieldTouched,
+                dirty,
+                setFieldValue,
+            }) => (
+                <React.Fragment>
+                    {!is_appstore && isMobile() && is_confirmation_visible && (
```

Comment:
why don't we use `is_mobile` from ui store?

Reply:
I was under the assumption that there is a card for refactoring and migration of Financial and trading assessment. Let me confirm on this